ts/src/index.ts

```diff
@@ -2,7 +2,12 @@ import { isBrowser } from "./utils/common.js";
 
 export { default as BN } from "bn.js";
 export * as web3 from "@solana/web3.js";
-export { default as Provider, getProvider, setProvider } from "./provider.js";
+export {
+  default as Provider,
+  getProvider,
+  setProvider,
+  Wallet,
```

Comment:
Wouldn't this conflict with the conditional export down below? Wallet is an interface while Node wallet is a class, but they're exported via the same namespace.